playground/mocks/idp/idx/data/select-factor-enroll.json

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
looks like we need to fix `factor-enroll-options.json` instead of creating new one?